internal/security/config/command/proxy/adduser/command.go

```diff
@@ -52,7 +52,7 @@ func NewCommand(
 	flagSet.StringVar(&dummy, "configDir", "", "") // handled by bootstrap; duplicated here to prevent arg parsing errors
 	flagSet.StringVar(&cmd.username, "user", "", "Username of the user to add")
 	flagSet.StringVar(&cmd.tokenTTL, "tokenTTL", DefaultTokenTTL, "Vault token created as a result of vault login lasts this long")
-	flagSet.StringVar(&cmd.jwtTTL, "jwtTTL", DefaultJWTTTL, "JWT created by vault identity provider lasts this long")
+	flagSet.StringVar(&cmd.jwtTTL, "jwtTTL", DefaultJWTTTL, "JWT created by vault identity provider lasts this long (_s, _m, _h, or _d, seconds if no unit)")
```

Comment:
Does the units also apply to `tokenTTL`?

Reply:
Yep.  Also updated tokenTTL likewise.